package.json

```diff
@@ -17,12 +17,12 @@
     "build": "babel src  --out-dir production",
     "start": "node production/server.js",
     "migrate:drop": "sequelize db:migrate:undo:all",
+    "seed:create": "npx sequelize-cli db:seed:all",
     "dev": "nodemon --exec babel-node src/server.js",
     "coverage": "nyc report --reporter=text-lcov | coveralls",
-    "pretest": "npm run migrate:drop && npm run migrate:up && npm run seed:up",
+    "pretest": "npm run migrate:drop && npm run migrate:up",
     "test": "export NODE_ENV=test && nyc --reporter=html --reporter=text mocha  --require  @babel/register src/tests/**/*.test.js --timeout 100000 --exit",
-    "migrate:up": "npx sequelize-cli db:migrate",
-    "seed:up": "npx sequelize-cli db:seed:all"
+    "migrate:up": "npx sequelize-cli db:migrate && npm run seed:create"
```

Comment:
Sometimes when you run migrations, you don't want to run seeders too.  Remember, dummy data is not allowed in production.

Reply:
If you need to run seeders in ci, add the script to travis config instead

Reply:
@adesege Ok solved now